server/templates/details.html.tmpl

```diff
@@ -148,3 +153,15 @@
   {{end}}
   </dl>
 {{end}}
+
+{{define "result-requires-permissions-details"}}
+  <b class="font-bold text-sm">This rule requires review from users with the following permissions</b>
+  <dl class="my-2">
+  {{range $k, $v := getPermissions .}}
+    <dt> {{$k}}</dt>
+    <dd>
+        <ul class="list-disc list-outside pl-6 py-2">{{range $v}}<li class="font-mono text-sm-mono">{{.}}</li>{{end}}</ul>
+    </dd>
+  {{end}}
+  </dl>
```

Comment:
Unlike the user/team/org section, `getPermissions` returns a `[]string` instead of a `map[string][]string`. I think you'll want to use a single `ul` rendered with one `range` expression for this, instead of the description list pattern.

Reply:
Fixed, copy-pasta laziness :p